components/balance/MultipleBalances.vue

```diff
@@ -92,6 +93,14 @@ const networkToPrefix = {
   statemint: 'ahp',
 }
 
+const networkToDisplayName = (chain: ChainType) => {
+  const networkToName = {
+    statemine: 'KusamaHub',
+    statemint: 'PolkadotHub',
+  }
+  return networkToName[chain] || chain
```

Comment:
Why not directly rename it here?

Reply:
Renaming it would be better, but there is some cache on the IdentityStore. I have solved the cache issue and rename it directly.